uiowa/uiowa
# Adjustable menu block title

Resolves https://github.com/uiowa/uiowa/issues/6164

The block title was originally hidden to avoid user confusion: https://github.com/uiowa/uiowa/pull/4667#issuecomment-1031874036.  Now that we are exposing it for users to use it as the mobile toggle button text, I have added an explanation of the title usage in the field description. 


# How to test

User story: As a site editor, I can adjust the section menu block title for the menu block and the mobile menu button label for the "Horizontal" menu option for the menu block.  

- `ddev blt frontend`
- `ddev blt ds --site=tippie.uiowa.edu`
- Verify that the mobile toggle menu button on https://tippie.uiowa.ddev.site/faculty-research/research-expertise-interests is "Section menu". 
- Edit the layout button and adjust the title field select on the menu block for the page.  Verify that you can set the parent menu item and other menu item options for the title. 
- Save the page and view in responsive mode and verify that the title has changed. 




**Before screenshots:**

<details>

<img width="682" alt="Screenshot 2023-04-18 at 3 43 14 PM" src="https://user-images.githubusercontent.com/1036433/232900548-67166e4c-d52d-4f15-8bde-3598d1d8e088.png">
<img width="517" alt="Screenshot 2023-04-18 at 3 43 06 PM" src="https://user-images.githubusercontent.com/1036433/232900550-75512fb0-bcb6-4061-aa90-29f88ab2c6b3.png">
<img width="468" alt="Screenshot 2023-04-18 at 3 42 53 PM" src="https://user-images.githubusercontent.com/1036433/232900552-46ef07df-0760-4f23-a1ae-cdc280764d6f.png">

</details>

**After screenshots:**

<details>
<img width="496" alt="Screenshot 2023-04-18 at 3 42 26 PM" src="https://user-images.githubusercontent.com/1036433/232900596-0232a98e-2d6a-4790-931d-60d9bab4d7bd.png">
<img width="487" alt="Screenshot 2023-04-18 at 3 42 08 PM" src="https://user-images.githubusercontent.com/1036433/232900599-cba18cd4-c63c-407c-8234-1386969e7118.png">
<img width="475" alt="Screenshot 2023-04-18 at 3 41 59 PM" src="https://user-images.githubusercontent.com/1036433/232900600-d45a811d-b615-42ec-a5ac-fe3926108146.png">

</details>

**Setting parent item as title example:**

<details>

<img width="1632" alt="Screenshot 2023-04-18 at 3 44 45 PM" src="https://user-images.githubusercontent.com/1036433/232900810-fef42bbb-bc9f-4e4c-a803-c4418b2079e7.png">
<img width="500" alt="Screenshot 2023-04-18 at 3 45 03 PM" src="https://user-images.githubusercontent.com/1036433/232900808-55712173-4112-4541-ba52-ddfe83286e4d.png">

</details>
